mobile-app/app/screens/AppNavigator/screens/Balances/components/EmptyPortfolio.tsx

```diff
@@ -0,0 +1,30 @@
+import { tailwind } from '@tailwind'
+import { ThemedText, ThemedView } from '@components/themed'
+import { View } from '@components'
+import { translate } from '@translations'
+import { EmptyWalletLight } from '../assets/EmptyWalletLight'
+import { EmptyWalletDark } from '../assets/EmptyWalletDark'
+import { useThemeContext } from '@shared-contexts/ThemeProvider'
+
+export function EmptyPortfolio (): JSX.Element {
+  const { isLight } = useThemeContext()
+  return (
+    <ThemedView
+      light={tailwind('bg-gray-50')}
+      style={tailwind('px-8 mt-8 pb-2 text-center')}
+      testID='empty_balances'
+    >
+      <View style={tailwind('items-center pb-4')}>
+        {
+          isLight ? <EmptyWalletLight /> : <EmptyWalletDark />
+        }
+      </View>
+      <ThemedText testID='empty_tokens_title' style={tailwind('text-lg pb-1 font-semibold text-center')}>
+        {translate('components/EmptyPortfolio', 'Nothing here yet')}
```

Comment:
Where do you get this copy from? I saw JJ issue screenshot is having "Empty portfolio"
Also found in [figma](https://www.figma.com/file/SDIpZciuxJlUf4xf8yheiY/PRDCT---Mobile-Light-Wallet?node-id=9531%3A300488)

Reply:
Modified to 'Empty portfolio'